mesheryctl/internal/cli/root/system/check.go

```diff
@@ -253,7 +254,9 @@ func (hc *HealthChecker) runKubernetesAPIHealthCheck() error {
 	//Check whether k8s client can be initialized
 	client, err := meshkitkube.New([]byte(""))
 	if err != nil {
-		failure++                 // increase failure count
+		if hc.context.Platform == "kubernetes" { // increase failure count
+			failure++
```

Comment:
How are we making use of failures counter? Are we printing it somewhere?

Reply:
check out Line 108.

Reply:
check out Line 108.

Reply:
Makes sense.